### DISCUSSION TITLE
`direct DDL is disabled` when running drizzle migrations

### DISCUSSION THREAD

**Discussion by ortonomy** [planetscale/discussion#572] [Q&A] [answered]
yo, I can't run my drizzle migrations against my database. I don't really care if it's production. It's my databsae. I can manage that. What gives

**Comment by hollylawly** [comment, marked as answer]
Disabling safe migrations should allow you to run direct ddl on your production branch
https://planetscale.com/docs/concepts/safe-migrations#how-to-disable-safe-migrations

**Reply by ortonomy** [reply]
Thanks!
